scripts/update_abi

```diff
@@ -1,20 +1,34 @@
 #!/usr/bin/env bash
 
-#Path to the autonity source code
-AUTONITY_PATH=../../../autonity-internal
 
+# Autonity node tag version
+# The releases are available at: 
+# https://github.com/autonity/autonity/releases
+AUTONITY_NODE_TAG=v0.10.1
```

Comment:
What is `AUTONITY_NODE_TAG` for? Seems to be unused and can risk getting out of sync of with `AUTONITY_NODE_COMMIT`.